### DISCUSSION TITLE
Add a “no-image-mode” to save data traffic on mobile networks

### DISCUSSION THREAD

**Discussion by etern** [jocmp/capyreader#461] [Feature Requests]
example configuration from telegram:
![image](https://github.com/user-attachments/assets/688f2b20-4ad5-4013-88e2-b89afe18e3ad)

**Comment by jocmp** [comment]
hey @etern. Thanks for the suggestion. Let me think about it. In your opinion would the images need to be downloadable on tap, or is it enough to hide the images completely?

**Reply by etern** [reply]
I think it's enough to skip downloading images completely.

As another reference, the browser I use(Via Browser), also have this configuration:

![image](https://github.com/user-attachments/assets/ef9a5bca-de23-483c-af49-9798953a27ab)

**Reply by jocmp** [reply]
Nice, that makes things a lot simpler.

**Reply by jocmp** [reply]
I added this in the [latest release](https://github.com/jocmp/capyreader/releases/tag/2024.11.1078-dev).
